spec/game_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'date'
+require_relative 'spec_helper'
+
+describe Game do
+  before :each do
+    @game = Game.new(true, '2022-10-11', '2019-11-11')
+  end
```

Comment:
From the project requirement, we should test all implemented methods. Well done you have tested some of the `game` methods but not all, please kindly test the `add_game` and the `list_game` methods in your `game_module.rb`